packages/aws-cdk/lib/init-templates/app/util/os.ts

```diff
@@ -12,9 +10,9 @@ export interface ShellOptions extends child_process.SpawnOptions {
  * Shell function which both prints to stdout and collects the output into a
  * string.
  */
-export async function shell(command: string[], options: ShellOptions = {}): Promise<string> {
-  debug(`Executing ${chalk.blue(renderCommandLine(command))}`);
-  const child = child_process.spawn(command[0], command.slice(1), {
+export async function shell(command: string[], options: ShellOptions): Promise<string> {
+  const child = child_process.spawn(command[0], renderArguments(command.slice(1)), {
+    shell: true,
```

Comment:
Why `shell: true` ?

What happens if we don't pass this command through the shell?